mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -37,7 +37,7 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	METHOD method_17696 (Lnet/minecraft/class_2248;Lnet/minecraft/class_1657;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Ljava/lang/Boolean;
 		ARG 2 world
 		ARG 3 pos
-	METHOD method_30010 removeStack (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
+	METHOD method_30010 onSlotClickUnsafe (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
```

Comment:
Why is this unsafe? Perhaps some javadoc explaining it could be useful here.

Reply:
This one is wrapped in a call from `onSlotClick` like how entity/block entity ticks are wrapped.

Reply:
Since this is a private method, I think the context is quite straightforward and doesn't need javadocs.

Reply:
If it's just the internal implementation of `onSlotClick`, naming it `onSlotClickImpl` or something similar would be better. Unless there's something that actually makes this method unsafe?

Reply:
Now I think should name it `internalOnSlotClick`